### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by andreidorin-oprea**
*Description of changes:*
Bump version to 21.10.0

### CODE REVIEW

**Comment by ubikusss** [comment]
@sshvans - FYI

File: scripts/modules/AzureUtils.psm1 [hunk 1]
```diff
@@ -1,162 +1,162 @@
...
-        return $max + 1
-
-    }
-    catch{
-        Write-Error -Exception $_.Exception -Message "Could not increment image version"
-        throw $_.Exception.Message
-    }
-}
+<#
+.Description
```
Comment (sshvans) [inline scripts/modules/AzureUtils.psm1:2]: Is this script about telemetry information?

Comment (andreidorin-oprea) [reply scripts/modules/AzureUtils.psm1]: Hi @sshvans, the script isn't about telemetry, just a common utils module we use across multiple scripts. However, we realized that we don't actually reference it in QS deployments and we removed it

Comment (sshvans) [reply scripts/modules/AzureUtils.psm1]: Thanks for removing it!

**Review by sshvans — COMMENTED** [review]
LGTM

**Review by sshvans — APPROVED** [review]

**Pull request merged**